CSV_README.md

```diff
@@ -45,10 +45,10 @@ We will attempt to accommodate your reasonable requests.
 
 Please submit your solution by emailing [hiring@truss.works](mailto:hiring@truss.works) with a link to a public git repository
 (Github is fine) that contains your code and a README.md that tells us
-how to build and run it. Your code will be run on either macOS 11.2+
-or Ubuntu 20.04 LTS, your choice. Note, if your development machine is
-running Windows, we suggest testing your response in a Docker image based on
-Ubuntu or Alpine linux.
+how to build and run it. **Your code will be run on your choice of macOS 11.2+ or Ubuntu 20.04 LTS.**
+
+> **Note**
+> If you develop your solution on Windows, be sure to test your response in [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install) under a distribution like Ubuntu or in a Docker image based on Ubuntu or Alpine linux.
```

Comment:
I'm conflicted about making testing a submission in linux/macOS optional (at least as far as what we ask for). I agree that we don't want folks to burn time setting something up that's not directly related to the projects; but on the other, if we have issues and find out they didn't test it on on linux/macOS, that hurts their submission, right?

My worry is that by being so permissive we're actually setting applicants up to fail should we encounter issues with their submission.

Reply:
I agree it hurts their submission if it doesn't work on linux/mac, but I feel like we should wait until we have the alternative we're talking about in slack before we make it required. 

Maybe this is a question for slack, but do we have numbers on how many submissions don't work for us because the submitter only tested on Windows? I feel like if the number is low, I'd rather it stay as a recommendation for now. 

To me, them writing clear documentation and pointing at the right docs for installation is more important than seeing how well they can do at setting up WSL or docker since many of us have to write docs, but not many have to do that type of setup.